authentication/views.py

```diff
@@ -336,6 +336,28 @@ def delete(self, request):
         return Response("Company Deleted Successfully", status.HTTP_200_OK)
 
 
+class ClientListView(generics.GenericAPIView, BaseUtils):
+    """
+    Return a list of registered clients
+    """
+    serializer_class = ClientSerializer
+    renderer_classes = (ClientJSONRenderer,)
+
+    def get_queryset(self):
+        return Client.active_objects.all()
+
+    def get(self, request):
+        serializer = self.serializer_class(self.get_queryset(), many=True)
+        response = {
+                "data": {
+                    "client_company": serializer.data,
```

Comment:
Hi @Rhytah , this is nicely done, Great job, I was wondering if you intended to have the data in this particular shape
![image](https://user-images.githubusercontent.com/12128153/63671556-71607e00-c7e8-11e9-8daf-9e5c5e35f1d2.png)

Whereby its 

```
{data: {data: client_company: [], message: ""}}
```

what do you think of 
```
{data: {client_company: [], message: ""}}
```


I was also thinking since it's a list of client companies then it should be `client_companies: []` and not `client_company: []`

Reply:
@Tevinthuku thanks for the feedback, I have made the changes to reflect data returned.